x-pack/plugins/cloud/server/assets/fullstory_library.js

```diff
@@ -9,5 +9,7 @@
\ No newline at end of file
+/* eslint-disable prettier/prettier,no-var,eqeqeq,new-cap,no-nested-ternary,no-use-before-define,no-sequences,block-scoped-var,one-var,
```

Comment:
Bad 3rd party code, bis.

Reply:
indeed! going through the list of errors and adding the specific rules to skip was painful 😅